### PR TITLE
Fix pipenv warning

### PR DESCRIPTION
**Pull request by patrick-codaio**
As [seen here](https://github.com/coda/packs-sdk/pull/1965#discussion_r864022046) pipenv does not want us to use `lock` anymore.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -39,7 +39,7 @@ _bootstrap-python-requirements:
 	# Ensure requirements.txt (used by Read the Docs) is in sync.
 	# Keep this separate from _bootstrap-python since the output changes based
 	# on the version of pipenv installed.
-	${PIPENV} lock -r > requirements.txt
+	${PIPENV} requirements > requirements.txt
```
Comment (dweitzman-codaio) [inline Makefile:42]: Could be nice to make this something in the spirit of `echo "# These requirements were autogenerated by 'make boostrap'" && ${PIPENV} requirements > requirements.txt` so the file still has a note about being auto-generated

**Review by dweitzman-codaio — APPROVED** [review]

File: requirements.txt [hunk 2]
```diff
@@ -1,34 +1,27 @@
-#
-# These requirements were autogenerated by pipenv
-# To regenerate from the project's Pipfile, run:
-#
-#    pipenv lock --requirements
-#
-
 -i https://pypi.org/simple
-bracex==2.2.1; python_version >= '3.6'
```
Comment (jonathan-codaio) [inline requirements.txt:9]: Does the loss of these `; python_version >= '3.6'` clauses matter? I suspect not.

Comment (patrick-codaio) [reply requirements.txt:9]: It didn't change the output of `make bs && make build`, so I guess not.

File: Makefile (hunk 1 above)
Comment (patrick-codaio) [reply Makefile]: Done!

**Pull request merged**